package.json

```diff
@@ -48,7 +48,7 @@
     "karma-mocha-reporter": "^2.2.0",
     "mocha": "^3.1.2",
     "native-promise-only": "^0.8.0-a",
-    "nyc": "^7.0.0",
+    "nyc": "^11.8.0",
```

Comment:
I'm assuming this is because the old version of `nyc` was using an older version of `lodash`?

Reply:
that's correct